app/controllers/api/cycles_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  class CyclesController < ApiApplicationController
+    before_action :set_curriculum, only: [:index]
+
+    def index
+      cycles = @curriculum.cycles
+      render json: cycles,
+             only: %i[id name learning_goals_description]
```

Comment:
Yes, I removed the unnecessary fields from the endpoint since we discussed and found that they weren't going to be used. I'll change the documentation.